core/templates/dev/head/app.js

```diff
@@ -415,40 +415,6 @@ oppia.factory('$exceptionHandler', ['$log', function($log) {
   };
 }]);
 
```

Comment:
Remove this line as 418 is also a new line.

Reply:
Done!

Reply:
Done!